tendrl/commons/message.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, priority, publisher, payload, job_id=None,
         else:
             self.caller = caller
         self.priority = priority
+        if not publisher:
+            publisher = NS.publisher_id
```

Comment:
what if the NS.publisher_id returns None, we have handled such a case here https://github.com/Tendrl/commons/blob/develop/tendrl/commons/message.py#L94,
we pass publisher like https://github.com/Tendrl/commons/blob/develop/tendrl/commons/__init__.py#L32